### PR TITLE
[tests-only] [full-ci] Update given step using sharingNg in coreApiShareUpdateToShares

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Description
Used sharingNG in given steps for sharing.

Suites covered:
- `coreApiShareUpdateToShares/updateShare.feature`

## Related Issue
- Part of https://github.com/owncloud/ocis/issues/8717

## How Has This Been Tested?
- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature [hunk 1]
```diff
@@ -36,8 +47,19 @@ Feature: updating shares to users and groups that have the same name
...
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Editor   |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | TMP      |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | group    |
+      | permissionsRole | Viewer   |
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature:61]:
```suggestion
      | permissionsRole | Editor   |
```
Role is updated from `Viewer` to `Viewer`

File: tests/acceptance/features/coreApiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature [hunk 2]
```diff
@@ -18,8 +18,19 @@ Feature: updating shares to users and groups that have the same name
...
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | group    |
+      | permissionsRole | Editor   |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | TMP      |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Viewer   |
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareUpdateToShares/updateShareGroupAndUserWithSameName.feature:32]:
```suggestion
      | permissionsRole | Editor   |
```

File: tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature [hunk 3]
```diff
@@ -170,7 +210,13 @@ Feature: sharing
     And user "Brian" has been added to group "grp1"
     And user "Carol" has been added to group "grp1"
     And user "Carol" has created folder "/FOLDER"
-    And user "Carol" has shared folder "/FOLDER" with group "grp1"
+    And user "Carol" has sent the following resource share invitation:
+      | resource        | FOLDER   |
+      | space           | Personal |
+      | sharee          | grp1     |
+      | shareType       | group    |
+      | permissionsRole | Viewer   |
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature:218]:
```suggestion
      | permissionsRole | Editor   |
```

File: tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature [hunk 4]
```diff
@@ -16,7 +16,13 @@ Feature: sharing
     And group "grp1" has been created
     And user "Brian" has been added to group "grp1"
     And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/textfile0.txt"
-    And user "Alice" has shared file "textfile0.txt" with group "grp1"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile0.txt |
+      | space           | Personal      |
+      | sharee          | grp1          |
+      | shareType       | group         |
+      | permissionsRole | Viewer        |
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature:24]:
```suggestion
      | permissionsRole | Editor        |
```

**Review by saw-jan — APPROVED** [review]

**Review by grgprarup — APPROVED** [review]
LGTM :+1:

File: tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature [hunk 5]
```diff
@@ -16,7 +16,13 @@ Feature: sharing
     And group "grp1" has been created
     And user "Brian" has been added to group "grp1"
     And user "Alice" has uploaded file "filesForUpload/textfile.txt" to "/textfile0.txt"
-    And user "Alice" has shared file "textfile0.txt" with group "grp1"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile0.txt |
+      | space           | Personal      |
+      | sharee          | grp1          |
+      | shareType       | group         |
+      | permissionsRole | Editor        |
```
Comment (grgprarup) [inline tests/acceptance/features/coreApiShareUpdateToShares/updateShare.feature:24]:
```suggestion
      | permissionsRole | File Editor        |
```

**Pull request merged**